ts/Dashboards/Components/KPIComponent.ts

```diff
@@ -36,9 +44,70 @@ const {
     merge
 } = U;
 
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ *
+ * Class that represents a KPI component.
+ *
+ */
 class KPIComponent extends Component {
-    public static charter?: typeof G;
 
+    /* *
+     *
+     *  Static functions
+     *
+     * */
+
+    /**
+     * Creates component from JSON.
+     *
+     * @param json
+     * Set of component options, used for creating the KPI component.
+     *
+     * @returns
+     * KPI component based on config from JSON.
+     */
+    public static fromJSON(json: KPIComponent.ClassJSON): KPIComponent {
+        const options = json.options;
+        const chartOptions =
+            options.chartOptions && JSON.parse(options.chartOptions);
+        const subtitle = JSON.parse(options.subtitle || '{}');
+        const title = options.title && JSON.parse(options.title);
+        const style = JSON.parse(options.style || '{}');
+        const thresholdColors = options.thresholdColors;
+        const value = options.value;
+        const threshold = options.threshold;
+        const valueFormat = options.valueFormat;
+
+        return new KPIComponent(
+            merge(options, {
+                chartOptions,
+                title,
+                subtitle,
+                style,
+                thresholdColors,
+                value,
+                threshold,
+                valueFormat
+            })
+        );
+    }
+
+    /* *
+     *
+     *  Static properties
+     *
+     * */
+
+    public static charter?: typeof G;
```

Comment:
Do we need this?

![image](https://user-images.githubusercontent.com/1453926/225003504-25a9f8bf-56a3-41b3-a567-59631f82a52a.png)